ui/src/app/applications/components/utils.tsx

```diff
@@ -656,13 +656,25 @@ export function renderResourceMenu(
                     {items.map((item, i) => (
                         <li
                             className={classNames('application-details__action-menu', {disabled: item.disabled})}
+                            tabIndex={0}
                             key={i}
                             onClick={e => {
                                 e.stopPropagation();
                                 if (!item.disabled) {
                                     item.action();
                                     document.body.click();
                                 }
+                            }}
+                            onKeyDown={e => {
+                                if (e.keyCode === 13 || e.key === 'Enter') {
+                                    e.stopPropagation();
+                                    if (!item.disabled) {
```

Comment:
If the current button is disabled, I think it shouldn't receive focus. Maybe we should just skip it.🤔

Reply:
This is indeed the better way!